recipes/sav_gol/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "sav_gol" %}
```

Comment:
I refrained from adding all the tools from the package, because many executables created by Tisean have very generic (i.e. "choose", "compare", "delay", "lazy", "project") or short and non-descriptive (i.e. "c1", "c2d", "d2", "pc") names, they might easily conflict with other programs installed in the system.
I believe having fine-grained control on what tools are installed helps avoiding name conflicts.
What do you suggest in such case?

Reply:
You can generate multiple packages from one recipe using the [`outputs`](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#outputs-section) section.

Rename the top level package to `tisean-split`, then separate the binaries into multiple outputs using the explicit file lists method.

Something like:

```yaml
package:
   name: tisean-split
   vesrion: {{ version }}

# build package as previously

# define how the above package is divided into subpackages
outputs:
  - name: tisean
    requirements:
       run:
         - {{ pin_subpackage( 'tisean-sav_gol', exact=True }}
         - {{ pin_subpackage( 'tisean-choose', exact=True }}
  - name: tisean-sav_gol
    files:
      - bin/sav_gol
    requirements:
      build:
        - {{ compiler('c') }}
  - name: tisean-choose
    files:
      - bin/choose
    requirements:
      build:
        - {{ compiler('c') }}
```

Reply:
Thank you, I didn't know about the `outputs` feature.

Reply:
Is it okay if I only include sav_gol as output for now and add more if someone requests it?

Reply:
You could do something like this:

```yaml
package:
   name: tisean-split
   vesrion: {{ version }}

# build package as previously

# define how the above package is divided into subpackages
outputs:
  - name: tisean
    files:
      - # everything except for sav_gol
    requirements:
       run:
         - {{ pin_subpackage( 'tisean-sav_gol', exact=True }}
  - name: tisean-sav_gol
    files:
      - bin/sav_gol
    requirements:
      build:
        - {{ compiler('c') }}

```
Then add additional outputs as requested.